docs/netlify.toml

```diff
@@ -202,3 +202,103 @@ to = "/docs/rasa/deploy/deploy-rasa/"
 [[redirects]]
 from = "/docs/rasa/action-server/deploy-action-server"
 to = "/docs/rasa/deploy/deploy-action-server/"
+
+[[redirects]]
+from = "/docs/rasa-x/changelog/rasa-x-changelog/"
+to = "/docs/rasa-enterprise/changelog/rasa-enterprise-changelog/"
+
+[[redirects]]
+from = "/docs/rasa-enterprise/changelog/rasa-x-changelog/"
+to = "/docs/rasa-enterprise/changelog/rasa-enterprise-changelog/"
```

Comment:
can you move all the rasa-enterprise redirects in the rasa-x repo? [here](https://github.com/RasaHQ/rasa-x/blob/main/docs/netlify.toml)